CMakeLists.txt

```diff
@@ -18,12 +18,25 @@ if (CMAKE_CXX_COMPILER_ID MATCHES "GNU")
   SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fpermissive")
 endif()
 
+# OpenMP
+find_package(OpenMP)
+if (OPENMP_FOUND)
+  set (CMAKE_C_FLAGS "${CMAKE_C_FLAGS} ${OpenMP_C_FLAGS}")
+  set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${OpenMP_CXX_FLAGS}")
+endif()
+
 # LBM-DEM settings
 OPTION(USE_DOUBLE_PRECISION "Use double precision" ON)
 
 # CMake Modules
 set(CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake" ${CMAKE_MODULE_PATH})
 
+# KOKKOS
+find_package(KOKKOS REQUIRED)
```

Comment:
But that code in `kokkos.cmake` looks a bit fragile (for example using `-Xcompiler`) so perhaps should be left out in case kokkos cmake support changes